scripts/setup.sh

```diff
@@ -175,6 +174,22 @@ if [ ! -d "$NODE_PATH" ]; then
   chmod -R 744 $NODE_MODULE_DIR
 fi
 
+# Download and install yarn.
+echo Checking if yarn is installed in $TOOLS_DIR
+if [ ! -d "$YARN_PATH" ]; then
+  echo Removing package-lock.json
+  rm -f package-lock.json
```

Comment:
What if package-lock.json does not exist?

Reply:
it'll fail quietly

Reply:
in the sense that nothing will happen if the file doesn't exist

Reply:
OK. Also, why do you need the -f? In general we should avoid using -f flags on users' systems...

Reply:
That's always why I need the `-f` which according to the man page: "ignore nonexistent files, never prompt"

Alternatively, I could do an if statement which avoids the `-f`

Reply:
Oh, I see. Hm, ok, let's leave it as-is. Thanks!

Reply:
sounds good